Source/Core/DolphinQt2/WiiUpdate.cpp

```diff
@@ -96,7 +96,10 @@ static WiiUtils::UpdateResult ShowProgress(QWidget* parent, Callable function, A
   auto* cancel_button = new QPushButton(QObject::tr("&Cancel"), parent);
   dialog.setCancelButton(cancel_button);
   Common::Flag was_cancelled;
-  QObject::disconnect(&dialog, &QProgressDialog::canceled, &dialog, &QProgressDialog::cancel);
+  // Disconnect the default cancel signal. We don't know if Qt connected it using the old
+  // or the new signal slot syntax, so we need to use the old syntax with a wildcard,
+  // which allows us to get rid of all slots in all cases.
+  QObject::disconnect(&dialog, SIGNAL(canceled()), nullptr, nullptr);
```

Comment:
I thought I was clear enough with the above comment :( it's not possible to use the new signal slot syntax here, because Qt uses the old style itself. Using the old style + a wildcard is the only way (AFAIK) to get rid of the default canceled() => cancel() connection, and it will keep working when Qt starts using the new style -- the new style won't work for this.